tests/scripts/all.sh

```diff
@@ -868,53 +703,16 @@ component_build_no_std_function () {
     make CC=gcc CFLAGS='-Werror -Wall -Wextra -O0'
 }
 
-component_build_no_ssl_srv () {
-    msg "build: full config except ssl_srv.c, make, gcc" # ~ 30s
-    scripts/config.pl full
-    scripts/config.pl unset MBEDTLS_SSL_SRV_C
-    make CC=gcc CFLAGS='-Werror -Wall -Wextra -O0'
-}
-
-component_build_no_ssl_cli () {
-    msg "build: full config except ssl_cli.c, make, gcc" # ~ 30s
-    scripts/config.pl full
-    scripts/config.pl unset MBEDTLS_SSL_CLI_C
-    make CC=gcc CFLAGS='-Werror -Wall -Wextra -O0'
-}
-
 component_build_no_sockets () {
+    # XXX Questionable we want to test this. May be redundant.
```

Comment:
I think this component should be removed. It's not as if there was a real risk that the crypto code would inadvertently start using sockets. This component was introduced by 009a2640f673447aedbad356337c0facc143b059 and at the time it was the only component with `-std=c99` (later `-pedantic`), but now we have others.

Reply:
Removed